test/dataset/test_hierarchical.py

```diff
@@ -136,38 +137,19 @@ def test_hts_to_dataset(mode: str):
             ds = hts.to_dataset(feat_dynamic_real=features_df)
     else:
         ds = hts.to_dataset(feat_dynamic_real=features_df)
+        entry = next(iter(ds))
```

Comment:
https://toolz.readthedocs.io/en/latest/_modules/toolz/itertoolz.html#first

Reply:
I can do that of course, but this is literally what `first` does